src/weights.jl

```diff
@@ -250,6 +249,66 @@ eweights(t::AbstractVector, r::AbstractRange, λ::Real) =
 
 # NOTE: No variance correction is implemented for exponential weights
 
+struct UnitWeights{S<:Integer, T<:Real} <: AbstractWeights{S, T, V where V<:Vector{T}}
+    len::S
```

Comment:
As I noted, better hardcode this as `Int` for simplicity.

Reply:
Still need to address this comment.

Reply:
Does it look better now?